lib/ansible/modules/service.py

```diff
@@ -35,8 +35,8 @@
             commands unless necessary.
           - V(restarted) will always bounce the service.
           - V(reloaded) will always reload.
-          - B(At least one of state and enabled are required.)
-          - Note that reloaded will start the service if it is not already started,
+          - B(At least one of O(state) and O(enabled) are required.)
```

Comment:
I am not sure if the `(` works inside `B()` - 

> Note that C(), B(), and I() do not allow escaping, and thus cannot contain the value ) as it always ends the formatting sequence. If you need to use ) inside C(), we recommend using V() instead; see the above section on semantic markup.

Reply:
Yup, I missed that here and in a couple of other places. Personally I find marking the options more relevant than marking them as bold, in fact  a couple of other modules do exactly that in their docs.

The other option would be to make it B()O()B()... which looks very cumbersome to me.

Or we leave it as is, lacking the semantic hint.

Reply:
Yup, I missed that and another one, just fixed them by removing the B(), on the assumption that it would be more beneficial to the reader to stress the semantic markup on the options rather than the bold typing for the whole phrase. Sentes like this are common across the Ansible ecosystem and - to my recollection - not usually marked as bold.